src/map/trade.cpp

```diff
@@ -486,7 +484,7 @@ void trade_tradeok(struct map_session_data *sd)
 	}
 
 	sd->state.deal_locked = 1;
-	clif_tradeitemok(sd, 0, 0);
+	clif_tradeitemok(*sd, 0, EXITEM_ADD_SUCCEED);
```

Comment:
Do we need to send -2 here now?

Reply:
This needs testing. Will test and adjust (or not) accordingly.

Reply:
This behavior matches official server.